tests/Agent/IntegrationTests/UnboundedServices/docker-compose.yml

```diff
@@ -131,4 +131,26 @@ services:
             # the connection string used by those tests
             - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD:-MysqlPassword}
         container_name: MySqlServer
-        
\ No newline at end of file
+
+    elastic:
+        image: elastic/elasticsearch:8.6.2
+        ports:
+            - "9200:9200"
+            - "9300:9300"
+        environment:
+            - discovery.type=single-node
+            - xpack.security.enabled=false
+            - xpack.security.enrollment.enabled=false
+            - xpack.security.http.ssl.enabled=false
+            - xpack.security.transport.ssl.enabled=false
+        container_name: ElasticServer
+
+    kibana:
+        image: kibana:8.6.2
```

Comment:
Formatting is off for the kibana sentry.  I think it will still work though.

Reply:
Yeah, it still works but I'll fix it anyway.